xbmc/settings/dialogs/CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
-set(SOURCES GUIDialogAudioDSPManager.cpp
```

Comment:
@Rechi 
Thanks for the hints. The fixes are [here](https://github.com/FernetMenta/xbmc/pull/422).